platforms/cdh5.12.0/docs/CDH-installation.md

```diff
@@ -562,12 +562,18 @@ com.splicemachine.hbase.BackupEndpointObserver</code>
 
 There are a few configuration modifications you might want to make:
 
+* [Enable automatically restart for HBase service](#enable-automatically-restart) if you want HBase recover automatically after some failures.
 * [Modify the Authentication Mechanism](#modify-the-authentication-mechanism) if you want to
   authenticate users with something other than the default *native
   authentication* mechanism.
 * [Modify the Log Location](#modify-the-logging-location) if you want your Splice Machine
   log entries stored somewhere other than in the logs for your region
   servers.
+  
+### Enable Automatically Restart
```

Comment:
Again, suggest Enable Automatic Restart throughout.  I'll stop marking it, but there are many references to it.